categories/marketing-and-video/rules-to-better-branding.md

```diff
@@ -10,6 +10,7 @@ index:
 - business-cards-branding
 - great-email-signatures
 - browser-add-branding
+- google-maps-profile
```

Comment:
@sethdailyssw  Looking at the other rules under Rules to Better Branding, i'm not sure this is an appropriate category for this rule. 

Feels much more at home under https://www.ssw.com.au/rules/rules-to-better-social-media-for-business/

Reply:
Done - removed from rules-to-better-branding and added to rules-to-better-social-media-for-business